packages/suite/src/components/wallet/InputError.tsx

```diff
@@ -28,21 +29,39 @@ type LinkProps = { url: Url };
 
 export type InputErrorProps = {
     button?: ButtonProps | LinkProps;
+    learnMoreAsTextUrl?: string;
     message?: string;
 };
 
-export const InputError = ({ button, message }: InputErrorProps) => (
-    <Wrapper>
-        <ContentWrapper>{message}</ContentWrapper>
-        {button &&
-            ('url' in button ? (
-                <LearnMoreButton url={button.url} />
-            ) : (
-                <ButtonWrapper>
-                    <Button size="tiny" variant="tertiary" onClick={button.onClick}>
-                        {button.text}
-                    </Button>
-                </ButtonWrapper>
-            ))}
-    </Wrapper>
-);
+export const InputError = ({ button, message, learnMoreAsTextUrl }: InputErrorProps) => {
+    const { translationString } = useTranslation();
+
+    return (
+        <Wrapper>
+            <ContentWrapper>
+                {message}{' '}
+                {/* To have Learn More link as a part of the text rather than a button. */}
+                {learnMoreAsTextUrl && (
+                    <Link
+                        href={learnMoreAsTextUrl}
+                        variant="nostyle"
+                        icon="externalLink"
+                        type="label"
+                    >
+                        {translationString('TR_LEARN_MORE')}
+                    </Link>
+                )}
+            </ContentWrapper>
+            {button &&
+                ('url' in button ? (
+                    <LearnMoreButton url={button.url} />
+                ) : (
```

Comment:
This is used just in one edge-case in the whole app. Please unify it that all occurences of learn more "button" are text url.

Reply:
Could you please refactor this component that in case
- `button` prop is available, render button. I would also like better name than `button`. It is not button, it is just wrapper of `text` and `onClick`. `buttonProps` would be better but maybe there is even better naming for that.
- `url` prop is available, render this learn more button

this `'url' in button` is mega hack and not a clean code, please fix it 🙏

Reply:
I agree, will refactor